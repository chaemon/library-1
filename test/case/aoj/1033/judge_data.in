22
1 2 24 2
3 1 10 8
5 6 19 6
8 11 8 5
17 1 17 7
15 8 15 5
2 15 12 5
14 7 15 7
14 8 14 7
13 8 14 8
13 9 13 8
1 13 6 13
2 13 4 15
2 13 2 12
2 12 5 9
5 10 5 7
12 15 12 11
10 13 14 13
10 15 14 11
10 11 14 15
14 12 10 8
12 10 13 9
1 2 24 2
22
1 2 24 2
3 1 10 8
5 6 19 6
8 11 8 5
17 1 17 7
15 8 15 5
2 15 12 5
14 7 15 7
14 8 14 7
13 8 14 8
13 9 13 8
1 13 6 13
2 13 4 15
2 13 2 12
2 12 5 9
5 10 5 7
12 15 12 11
10 13 14 13
10 15 14 11
10 11 14 15
14 12 10 8
12 10 13 9
1 2 8 5
22
1 2 24 2
3 1 10 8
5 6 19 6
8 11 8 5
17 1 17 7
15 8 15 5
2 15 12 5
14 7 15 7
14 8 14 7
13 8 14 8
13 9 13 8
1 13 6 13
2 13 4 15
2 13 2 12
2 12 5 9
5 10 5 7
12 15 12 11
10 13 14 13
10 15 14 11
10 11 14 15
14 12 10 8
12 10 13 9
1 2 12 5
22
1 2 24 2
3 1 10 8
5 6 19 6
8 11 8 5
17 1 17 7
15 8 15 5
2 15 12 5
14 7 15 7
14 8 14 7
13 8 14 8
13 9 13 8
1 13 6 13
2 13 4 15
2 13 2 12
2 12 5 9
5 10 5 7
12 15 12 11
10 13 14 13
10 15 14 11
10 11 14 15
14 12 10 8
12 10 13 9
1 2 1 13
22
1 2 24 2
3 1 10 8
5 6 19 6
8 11 8 5
17 1 17 7
15 8 15 5
2 15 12 5
14 7 15 7
14 8 14 7
13 8 14 8
13 9 13 8
1 13 6 13
2 13 4 15
2 13 2 12
2 12 5 9
5 10 5 7
12 15 12 11
10 13 14 13
10 15 14 11
10 11 14 15
14 12 10 8
12 10 13 9
1 2 14 15
22
1 2 24 2
3 1 10 8
5 6 19 6
8 11 8 5
17 1 17 7
15 8 15 5
2 15 12 5
14 7 15 7
14 8 14 7
13 8 14 8
13 9 13 8
1 13 6 13
2 13 4 15
2 13 2 12
2 12 5 9
5 10 5 7
12 15 12 11
10 13 14 13
10 15 14 11
10 11 14 15
14 12 10 8
12 10 13 9
15 8 12 11
22
1 2 24 2
3 1 10 8
5 6 19 6
8 11 8 5
17 1 17 7
15 8 15 5
2 15 12 5
14 7 15 7
14 8 14 7
13 8 14 8
13 9 13 8
1 13 6 13
2 13 4 15
2 13 2 12
2 12 5 9
5 10 5 7
12 15 12 11
10 13 14 13
10 15 14 11
10 11 14 15
14 12 10 8
12 10 13 9
15 8 5 7
22
1 2 24 2
3 1 10 8
5 6 19 6
8 11 8 5
17 1 17 7
15 8 15 5
2 15 12 5
14 7 15 7
14 8 14 7
13 8 14 8
13 9 13 8
1 13 6 13
2 13 4 15
2 13 2 12
2 12 5 9
5 10 5 7
12 15 12 11
10 13 14 13
10 15 14 11
10 11 14 15
14 12 10 8
12 10 13 9
14 8 5 6
22
1 2 24 2
3 1 10 8
5 6 19 6
8 11 8 5
17 1 17 7
15 8 15 5
2 15 12 5
14 7 15 7
14 8 14 7
13 8 14 8
13 9 13 8
1 13 6 13
2 13 4 15
2 13 2 12
2 12 5 9
5 10 5 7
12 15 12 11
10 13 14 13
10 15 14 11
10 11 14 15
14 12 10 8
12 10 13 9
24 2 10 15
22
1 2 24 2
3 1 10 8
5 6 19 6
8 11 8 5
17 1 17 7
15 8 15 5
2 15 12 5
14 7 15 7
14 8 14 7
13 8 14 8
13 9 13 8
1 13 6 13
2 13 4 15
2 13 2 12
2 12 5 9
5 10 5 7
12 15 12 11
10 13 14 13
10 15 14 11
10 11 14 15
14 12 10 8
12 10 13 9
5 10 14 7
18
1 1 4 1
5 1 11 1
1 5 3 5
5 5 11 5
1 8 11 8
3 6 5 6
3 4 5 4
1 8 1 1
11 1 11 8
3 6 3 4
5 4 5 6
3 3 6 3
3 3 4 2
3 2 4 2
3 2 4 1
5 2 6 3
5 2 6 2
6 2 5 1
1 1 11 8
18
1 1 4 1
5 1 11 1
1 5 3 5
5 5 11 5
1 8 11 8
3 6 5 6
3 4 5 4
1 8 1 1
11 1 11 8
3 6 3 4
5 4 5 6
3 3 6 3
3 3 4 2
3 2 4 2
3 2 4 1
5 2 6 3
5 2 6 2
6 2 5 1
1 1 11 1
18
1 1 4 1
5 1 11 1
1 5 3 5
5 5 11 5
1 8 11 8
3 6 5 6
3 4 5 4
1 8 1 1
11 1 11 8
3 6 3 4
5 4 5 6
3 3 6 3
3 3 4 2
3 2 4 2
3 2 4 1
5 2 6 3
5 2 6 2
6 2 5 1
5 6 4 1
18
1 1 4 1
5 1 11 1
1 5 3 5
5 5 11 5
1 8 11 8
3 6 5 6
3 4 5 4
1 8 1 1
11 1 11 8
3 6 3 4
5 4 5 6
3 3 6 3
3 3 4 2
3 2 4 2
3 2 4 1
5 2 6 3
5 2 6 2
6 2 5 1
4 2 11 1
18
2 11 2 3
3 11 3 3
4 11 4 1
5 11 5 1
6 8 6 7
7 8 7 7
8 10 8 3
9 10 9 2
10 10 10 1
11 11 11 0
0 10 15 10
4 9 12 9
6 8 7 8
6 7 7 7
1 6 15 6
7 5 14 5
2 2 7 2
8 2 14 2
2 11 14 2
18
2 11 2 3
3 11 3 3
4 11 4 1
5 11 5 1
6 8 6 7
7 8 7 7
8 10 8 3
9 10 9 2
10 10 10 1
11 11 11 0
0 10 15 10
4 9 12 9
6 8 7 8
6 7 7 7
1 6 15 6
7 5 14 5
2 2 7 2
8 2 14 2
2 11 7 7
18
2 11 2 3
3 11 3 3
4 11 4 1
5 11 5 1
6 8 6 7
7 8 7 7
8 10 8 3
9 10 9 2
10 10 10 1
11 11 11 0
0 10 15 10
4 9 12 9
6 8 7 8
6 7 7 7
1 6 15 6
7 5 14 5
2 2 7 2
8 2 14 2
6 8 6 7
18
2 11 2 3
3 11 3 3
4 11 4 1
5 11 5 1
6 8 6 7
7 8 7 7
8 10 8 3
9 10 9 2
10 10 10 1
11 11 11 0
0 10 15 10
4 9 12 9
6 8 7 8
6 7 7 7
1 6 15 6
7 5 14 5
2 2 7 2
8 2 14 2
8 2 7 2
18
2 11 2 3
3 11 3 3
4 11 4 1
5 11 5 1
6 8 6 7
7 8 7 7
8 10 8 3
9 10 9 2
10 10 10 1
11 11 11 0
0 10 15 10
4 9 12 9
6 8 7 8
6 7 7 7
1 6 15 6
7 5 14 5
2 2 7 2
8 2 14 2
2 2 15 10
18
2 11 2 3
3 11 3 3
4 11 4 1
5 11 5 1
6 8 6 7
7 8 7 7
8 10 8 3
9 10 9 2
10 10 10 1
11 11 11 0
0 10 15 10
4 9 12 9
6 8 7 8
6 7 7 7
1 6 15 6
7 5 14 5
2 2 7 2
8 2 14 2
14 2 7 8
21
2 1 3 2
3 2 2 2
2 2 3 3
3 3 4 2
4 2 5 1
5 1 9 5
9 5 5 9
5 9 5 8
5 8 6 7
6 7 5 6
5 6 6 6
6 6 7 5
7 5 1 5
1 5 1 6
1 6 2 6
2 6 2 7
2 7 3 7
3 7 3 8
3 8 5 10
5 10 6 9
6 9 6 11
2 1 6 11
10
4 3 11 3
8 5 8 2
8 2 9 6
7 8 7 3
3 5 7 5
3 5 15 7
15 7 10 10
1 6 7 6
2 5 6 8
6 11 6 7
11 3 9 6
10
4 3 11 3
8 5 8 2
8 2 9 6
7 8 7 3
3 5 7 5
3 5 15 7
15 7 10 10
1 6 7 6
2 5 6 8
6 11 6 7
11 3 15 7
10
4 3 11 3
8 5 8 2
8 2 9 6
7 8 7 3
3 5 7 5
3 5 15 7
15 7 10 10
1 6 7 6
2 5 6 8
6 11 6 7
11 3 1 6
10
4 3 11 3
8 5 8 2
8 2 9 6
7 8 7 3
3 5 7 5
3 5 15 7
15 7 10 10
1 6 7 6
2 5 6 8
6 11 6 7
3 5 10 10
10
4 3 11 3
8 5 8 2
8 2 9 6
7 8 7 3
3 5 7 5
3 5 15 7
15 7 10 10
1 6 7 6
2 5 6 8
6 11 6 7
3 5 1 6
10
4 3 11 3
8 5 8 2
8 2 9 6
7 8 7 3
3 5 7 5
3 5 15 7
15 7 10 10
1 6 7 6
2 5 6 8
6 11 6 7
9 6 1 6
10
4 3 11 3
8 5 8 2
8 2 9 6
7 8 7 3
3 5 7 5
3 5 15 7
15 7 10 10
1 6 7 6
2 5 6 8
6 11 6 7
1 6 6 11
10
4 3 11 3
8 5 8 2
8 2 9 6
7 8 7 3
3 5 7 5
3 5 15 7
15 7 10 10
1 6 7 6
2 5 6 8
6 11 6 7
10 10 6 11
10
4 3 11 3
8 5 8 2
8 2 9 6
7 8 7 3
3 5 7 5
3 5 15 7
15 7 10 10
1 6 7 6
2 5 6 8
6 11 6 7
3 5 7 8
10
4 3 11 3
8 5 8 2
8 2 9 6
7 8 7 3
3 5 7 5
3 5 15 7
15 7 10 10
1 6 7 6
2 5 6 8
6 11 6 7
8 5 6 11
0
