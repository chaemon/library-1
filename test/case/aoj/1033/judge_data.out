0.00000000
180.00000000
135.00000000
180.00000000
225.00000000
450.00000000
360.00000000
270.00000000
225.00000000
540.00000000
90.00000000
180.00000000
360.00000000
540.00000000
270.00000000
-1
0.00000000
360.00000000
180.00000000
-1
1845.00000000
104.03624347
170.53767779
180.00000000
139.57392126
170.53767779
189.46232221
90.00000000
525.29644818
80.53767779
466.26020471
